### PR TITLE
fix incorrect p90/p95/p99

### LINKED ISSUE #670: p95, p90 return incorrect result
**Describe what's wrong**

1. run the below two SQLs:
```SQL
with cte as (select array_join([100, 200, 300, 400, 500]) as f) select p95(f) from cte;

with cte as (select array_join([100, 200, 300, 400, 500]) as f) select p90(f) from cte;
```
2. both SQL return 300 as result

Expected Result:
p90: 460
p95:  479.99999999999994

**How to reproduce**

**Error message and/or stacktrace**

**Additional context**

### PR DESCRIPTION
**Pull request by yl-lisen**
PR checklist:
- Did you run ClangFormat ?
- Did you separate headers to a different section in existing community code base ?
- Did you surround `proton: starts/ends` for new code in existing community code base ?

Please write user-readable short description of the changes:
This close #670

### CODE REVIEW

File: src/AggregateFunctions/AggregateFunctionQuantile.h [hunk 1]
```diff
@@ -87,7 +87,7 @@ class AggregateFunctionQuantile final
         {
             assert(this->levels.size() == 1);  /// default: 0.5
             /// Make level: 'p90' -> 0.90  'p95' -> 0.95  'p99' -> 0.99
-            this->levels.levels[0] = std::stod(fmt::format("0.{}", this->getName().substr(1)));
+            this->level = this->levels.levels[0] = std::stod(fmt::format("0.{}", this->getName().substr(1)));
```
Comment (yokofly) [inline src/AggregateFunctions/AggregateFunctionQuantile.h:90]:
```suggestion
            this->levels.levels[0] = std::stod(fmt::format("0.{}", this->getName().substr(1)));
            this->level = this->levels.levels[0];
```

Comment (yokofly) [reply src/AggregateFunctions/AggregateFunctionQuantile.h:90]: is this clear?

Comment (yl-lisen) [reply src/AggregateFunctions/AggregateFunctionQuantile.h:90]: I think the both are ok

Comment (yl-lisen) [reply src/AggregateFunctions/AggregateFunctionQuantile.h:90]: it's not a complex code

**Review by yokofly — APPROVED** [review]

**Pull request merged**